task 'test:server' failed after upgrade to Kibana 5.6.7

Hello,

Context: I'm migrating custom Kibana plugins from Kibana 5.2.2 to Kibana 5.6.7.
I updated my `package.json`, upgraded my code, tested with `npm run start` and everything is fine.
I'm using now "@elastic/plugin-helpers": "8.1.3" which seems to be the latest version compatible with my setup.
I did not run the "template-kibana-plugin" again to do this migration.

What's happening: `npm run test:server` fails with following error:

```
Error: Cannot find module 'D:\Users\xxxxxx\src\kibana\test\mocha_setup.js'
```

My understanding: the thing is that `mocha_setup.js` does not belong to Kibana 5.6.7 anymore. It seems that there were plans to move scripts ([kibana#11095](https://github.com/elastic/kibana/issues/11095)) with impacts on makelogs and mocha npm scripts ([kibana #11665](https://github.com/elastic/kibana/pull/11665)) which were backported to 5.x.
Plugin Helpers still reference this `mocha_setup.js` script in this version ([link](https://github.com/elastic/kibana-plugin-helpers/blob/master/tasks/test/server/test_server_action.js#L9): 
```js
const mochaSetupJs = resolve(plugin.kibanaRoot, 'test/mocha_setup.js');
```
I think the `test_server_action.js` file needs a patch.

I know my setup is deprecated, but is there any chance that this gets fixed? Any idea of a workaround?

--- a/tasks/test/server/test_server_action.js
+++ b/tasks/test/server/test_server_action.js
@@ -3,10 +3,23 @@ const delimiter = require('path').delimiter;
 const execFileSync = require('child_process').execFileSync;
 const winCmd = require('../../../lib/win_cmd');
 
+function canRequire(path) {
+  try {
+    require.resolve(path);
+    return true;
+  } catch (error) {
+    if (error.code !== 'MODULE_NOT_FOUND') {
+      throw error;
+    }
+    return false;
+  }
+}
+
 module.exports = function (plugin, run, options) {
   options = options || {};
   const kibanaBins = resolve(plugin.kibanaRoot, 'node_modules/.bin');
-  const mochaSetupJs = resolve(plugin.kibanaRoot, 'test/mocha_setup.js');
+  const mochaSetupScript = resolve(plugin.kibanaRoot, 'test/mocha_setup.js');
+  const babelRegisterScript = resolve(plugin.kibanaRoot, 'src/optimize/babel/register');
   let testPaths = plugin.serverTestPatterns;
 
   // allow server test files to be overridden
@@ -16,7 +29,7 @@ module.exports = function (plugin, run, options) {
 
   const fullCmd = resolve(plugin.kibanaRoot, 'node_modules', '.bin', 'mocha');
   const cmd = winCmd(fullCmd);
-  const args = ['--require', mochaSetupJs].concat(testPaths);
+  const args = ['--require', canRequire(mochaSetupScript) ? mochaSetupScript : babelRegisterScript].concat(testPaths);
   const path = `${kibanaBins}${delimiter}${process.env.PATH}`;
 
   execFileSync(cmd, args, {

